wrapper.r2py

```diff
@@ -457,7 +457,11 @@ def _check_func_args(name, arg_definition, args):
       if "any" in expected_types:
         match = True
       else:
-        match = isinstance(args[index], expected_types)
+        # handling 'bool' seperately
```

Comment:
From a comment standpoint, you need to explain why this is happening.  Not just what you're doing.  So, some context (similar to the issue you filed) is relevant here.

Reply:
Added comment explaining this. Please let me know if it needs to be elaborate or made more concise.